posthog/api/feature_flag.py

```diff
@@ -339,6 +344,23 @@ def evaluation_reasons(self, request: request.Request, **kwargs):
 
         return Response(flags_with_evaluation_reasons)
 
+    @action(methods=["POST"], detail=False)
+    def user_blast_radius(self, request: request.Request, **kwargs):
```

Comment:
might make sense to cache this to some degree like we do with insights. if someone is adding and removing conditions in rapid succession we might dispatch too many queries

Reply:
Makes sense. I cached the total, but didn't consider cats rolling on the keyboard 😂 - will add